webpack/scenes/AlternateContentSources/Create/Steps/AcsUrlPaths.js

```diff
@@ -71,6 +90,15 @@ const AcsUrlPaths = () => {
             onChange={checked => setVerifySSL(checked)}
           />
         </FormGroup>
+        {verifySSL && acsType === 'rhui' &&
+          <>
+            {__('Generate certificates for the chosen repositories')}
```

Comment:
I'd instead put this on the "Credentials" part of the wizard under the "Content credentials" bullet.

I think we could make the "Content credentials" bullet the default selection too. I don't know when someone would connect to rhui with basic auth.  In fact -- I'm pretty sure we can remove the basic auth section ("Manual authentication") entirely.

![image](https://user-images.githubusercontent.com/14796566/194956487-f48448f0-0ead-4d95-b9f5-92b1ceaf84e5.png)